workspace_tools/export/atmelstudio.py

```diff
@@ -0,0 +1,69 @@
+"""
+mbed SDK
+Copyright (c) 2011-2015 ARM Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from exporters import Exporter
+from os.path import splitext, basename, dirname
+
+
+class AtmelStudio(Exporter):
+    NAME = 'AtmelStudio'
+    TOOLCHAIN = 'GCC_ARM'
+
+    TARGETS = [
+        'SAMD21J18A',
+        'SAMR21G18A',
+        'SAMD21G18A',
+        'SAML21J18A',
+    ]
+
+    DOT_IN_RELATIVE_PATH = True
+
+    def generate(self):
+
+        source_files = []
+        dirs = []
+        for r_type in ['s_sources', 'c_sources', 'cpp_sources']:
+            r = getattr(self.resources, r_type)
+            if r:
+                for source in r:
+                    source_files.append(source[2:])
+                    dirs.append(dirname(source[2:]))
+
+        source_folders = []
+        for e in dirs:
+            if e and e not in source_folders:
+                source_folders.append(e)
+
+        libraries = []
+        for lib in self.resources.libraries:
+            l, _ = splitext(basename(lib))
+            libraries.append(l[3:])
+
+        ctx = {
+            'target': self.target,
+            'name': self.program_name,
+            'source_files': source_files,
+            'source_folders': source_folders,
+            'object_files': self.resources.objects,
+            'include_paths': self.resources.inc_dirs,
+            'library_paths': self.resources.lib_dirs,
+            'linker_script': self.resources.linker_script,
+            'libraries': libraries,
+            'symbols': self.get_symbols()
+        }
+        target = self.target.lower()
+        self.gen_file('atmelstudio6_2.atsln.tmpl', ctx, '%s.atsln' % self.program_name)
```

Comment:
Since Atmel studio 6.2 has much wider user base we stick to 6.2 project template. As you pointed out, 6.2 projects can be opened in 7.0 Studio so we thought there is no need for separate template for 7.0. 
Further the mbed online ide export option just lists single entry for each Toolchain like IAR, Keil etc. without any IDE version.